pkg/plugin/util/util.go

```diff
@@ -110,3 +116,67 @@ func SelfLinkToCRDName(selfLink string) string {
 		return components[pluralIndex]
 	}
 }
+
+func RetrieveStorageClassMapping(config *rest.Config, veleroNs string, storageClassMapping map[string]string, logger logrus.FieldLogger) error {
+	var err error
+	if config == nil {
+		config, err = rest.InClusterConfig()
+		if err != nil {
+			logger.WithError(err).Errorf("Failed to get k8s inClusterConfig")
+			return errors.Wrap(err, "Could not retrieve in-cluster config")
+		}
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		logger.WithError(err).Errorf("Failed to get k8s clientset from the given config: %v", config)
+		return err
+	}
+	labelSelector := metav1.LabelSelector{MatchLabels: map[string]string{"velero.io/plugin-config":"", "velero.io/change-storage-class":"RestoreItemAction"}}
+	configMaps, err := clientset.CoreV1().ConfigMaps(veleroNs).List(context.TODO(), metav1.ListOptions{
+		LabelSelector: labels.Set(labelSelector.MatchLabels).String(),
+	})
+	if err != nil {
```

Comment:
Change to check NotFound error, if no config map found, we'll just return no need to update storage class name in pvc.

Reply:
Have you tested this?  What will return if there are no configmaps?

Reply:
Please confirm this case was covered in the description, so that we can make sure there is no regression introduced.

Reply:
If no eligible configMap exists, clientset.CoreV1().ConfigMaps(veleroNs).List will not return error, it will return a configMap with 0 Items in it, which is checked in:
```
if len(configMaps.Items) == 0 {
	logger.Info("No config map for storage class mapping exists.")
	return nil, nil
}
```
And this case was manually tested:
```
time="2020-11-05T02:27:43Z" level=info msg="Retrieving storage class mapping information from configMap" cmd=/plugins/velero-plugin-for-vsphere logSource="/go/src/github.com/vmware-tanzu/velero-plugin-for-vsphere/pkg/plugin/restore_pvc_action_plugin.go:143" pluginName=velero-plugin-for-vsphere restore=velero/zqzztoj
time="2020-11-05T02:27:43Z" level=info msg="No config map for storage class mapping exists." cmd=/plugins/velero-plugin-for-vsphere logSource="/go/src/github.com/vmware-tanzu/velero-plugin-for-vsphere/pkg/plugin/util/util.go:154" pluginName=velero-plugin-for-vsphere restore=velero/zqzztoj
time="2020-11-05T02:27:43Z" level=info msg="Found Backup Storage Location default for the Backup vihmxxt" cmd=/plugins/velero-plugin-for-vsphere logSource="/go/src/github.com/vmware-tanzu/velero-plugin-for-vsphere/pkg/utils/utils.go:239" pluginName=velero-plugin-for-vsphere restore=velero/zqzztoj
```